etna/transforms/timestamp/date_flags.py

```diff
@@ -6,12 +6,11 @@
 import numpy as np
 import pandas as pd
 
-from etna.transforms.base import DymmyInColumnMixin
 from etna.transforms.base import FutureMixin
 from etna.transforms.base import Transform
 
 
-class DateFlagsTransform(Transform, FutureMixin, DymmyInColumnMixin):
+class DateFlagsTransform(Transform, FutureMixin):
```

Comment:
Where do we store information about `in_column` of this class now?

Reply:
I set the default value for in_column attribute in the base class, in the next tasks all the transforms will be inherited from NewTransform